db/db_local/backup.go

```diff
@@ -79,7 +105,7 @@ func takeBackup(ctx context.Context, config *dbStartConfig) error {
 }
 
 // startDatabaseInLocation starts up the postgres binary in a specific installation directory
```

Comment:
does this always start PG12 instance? presumably as it returns `pg12RunningInfo` - in which case say so